### PR TITLE
Add socials

### PR DESCRIPTION
**Pull request by GresosJ**

### CODE REVIEW

**Review by ruilopesm — COMMENTED** [review]
image
Icons are being rendered too big 😅
This is a known issue when working with Next.js + fontawesome icons, btw.
I would suggest you to use [`react-icons`](https://react-icons.github.io/react-icons/).

**Comment by GresosJ** [comment]
But is that a really big problem?

File: src/components/Hero.jsx [hunk 1]
```diff
@@ -74,38 +68,38 @@ export function Hero() {
...
                 href="https://www.instagram.com/jordiuminho"
                 target="_blank"
                 rel="noreferrer"
               >
-                <FontAwesomeIcon icon={faInstagram} />
+                <FaInstagram />
               </a>
               <a
                 className="mr-5 w-5 hover:opacity-50"
                 href="https://www.instagram.com/jordiuminho"
```
Comment (ruilopesm) [inline src/components/Hero.jsx:90]: This should link to LinkedIn 🙏🏼

Comment (GresosJ) [reply src/components/Hero.jsx:90]: We dont have the likedin created yet

**Review by ruilopesm — APPROVED** [review]

**Review by EnzoVieira — APPROVED** [review]

**Pull request merged**